images/virtualization-artifact/pkg/apiserver/registry/vm/storage/util.go

```diff
@@ -0,0 +1,28 @@
+package storage
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+)
+
+func nameFor(fs fields.Selector) (string, error) {
+	if fs == nil {
+		fs = fields.Everything()
+	}
+	name := ""
+	if value, found := fs.RequiresExactMatch("metadata.name"); found {
```

Comment:
name, found := fs.RequiresExactMatch("metadata.name")
if !fs.Empty() {
	return "", fmt.Errorf("field label not supported: %s", fs.Requirements()[0].Field)
}

Reply:
its ok